packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -481,19 +481,7 @@ export class ConnectionManager implements IConnectionManager {
                     connection = undefined;
                 }
             } catch (origError) {
-                if (typeof origError === "object" && origError !== null &&
-                    origError?.errorType === DeltaStreamConnectionForbiddenError.errorType) {
-                    connection = new NoDeltaStream();
-                    requestedMode = "read";
-                    break;
-                }
-
-                // Socket.io error when we connect to wrong socket, or hit some multiplexing bug
-                if (!canRetryOnError(origError)) {
-                    const error = normalizeError(origError, { props: fatalConnectErrorProp });
-                    this.props.closeHandler(error);
-                    throw error;
-                }
+                const error = normalizeError(origError);
```

Comment:
I do not think it's right to push DeltaStreamConnectionForbiddenError.errorType case to go through logNetworkFailure(). If you want to add logging, add specific logging statement for just that case with unique event name - this seems more appropriate.